doc/UsingLibical.md

```diff
@@ -11,23 +11,10 @@ and protocol data units. The iCalendar specification describes how
 calendar clients can communicate with calendar servers so users can
 store their calendar data and arrange meetings with other users.
 
-Libical implements [RFC5545][], [RFC5546][], [RFC7529][];
-the CalDav scheduling extensions in [RFC6638][];
-the iCalendar extensions in [RFC7986][], [RFC9073][], [RFC9074][];
-and some of [RFC6047][].
-
+Libical implements multiple [RFC calendar standards](@ref about).
```

Comment:
An alternative would look like this:

```suggestion
Libical implements multiple RFC calendar standards, listed in the [README](../README.md).
```

We could then remove the `{#about}` tag from the README.

Reply:
Thanks .
I took the first idea and modified it a bit by add a new "RFCs" section to the README.me

new version is up at http://libical.github.io/libical/apidocs/index.html